brainzutils/cache.py

```diff
@@ -273,19 +253,15 @@ def gen_key(key, *attributes):
 
 def _prep_dict(dictionary, namespace=None, encode=True):
     """Wrapper for _prep_key and _encode_val functions that works with dictionaries."""
-    if namespace:
-        namespace_and_version = _append_namespace_version(namespace)
-    else:
-        namespace_and_version = None
-    return {_prep_key(key, namespace_and_version): _encode_val(value) if encode else value
+    return {_prep_key(key, namespace): _encode_val(value) if encode else value
             for key, value in dictionary.items()}
 
 
-def _prep_key(key, namespace_and_version=None):
+def _prep_key(key, namespace):
```

Comment:
I had a quick look, but do we have tests for just `prep_key`? `test_single_dict_fancy` kind of does something (but this also allows non-ascii key values, so...)

Reply:
Yeah, I looked as well. We don't have tests for just prep_key.